src/controller/user.controller.ts

```diff
@@ -48,9 +48,9 @@ const updateUser = async (req: Request, res: Response) => {
 const deleteUser = async (req: Request, res: Response) => {
   try {
     const userId = req.params.userId;
-    const deleteUser = await UserService.deleteUser(userId);
+    await UserService.deleteUser(userId);
 
-    res.status(200).json(deleteUser);
+    res.status(204).end();
```

Comment:
Should it send something back to the client so the client knows it has been deleted successfully, For example, a message?